### DISCUSSION TITLE
Interpretability of Segmentation UNet

### DISCUSSION THREAD

**Discussion by joho84** [Project-MONAI/MONAI#3005] [Q&A]
Hi!
First of all, thank you very much for your work and the great support you offer!
I have a question concerning the interpretability of a segmentation Unet. I am trying to get some more insight into the functioning of my Unet trained on multi-channel data. In especially I would be interested in which channels and which voxels contribute the most to the decision of the trained Unet for the segmentation of a specific class. Something similar to this: https://arxiv.org/abs/2002.11434
Is there already a way to achieve this in monai? Or could you point me in the right direction to adapt the existing interpretability code to a segmentation task?
Thank you in advance!

**Comment by rijobro** [comment]
We have GradCAM implemented for classification tasks. AFAIK, we don't have anything for segmentation, though I'm sure it wouldn't be much work to extend it to segmentation.

**Reply by joho84** [reply]
@rijobro Thank you for your quick answer! 
Any chance you could point me in the right direction for this? To be honest I am not sure where to start with the code adaptation. Or are you planning such an extension anyway?
Thank you in advance!

**Reply by rijobro** [reply]
@wyli has created an issue Feature Request from your Discussion: https://github.com/Project-MONAI/MONAI/issues/3032.
